sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/Ecdsa.java

```diff
@@ -52,10 +52,11 @@ public boolean verify(byte[] digest, byte[] signature) throws GeneralSecurityExc
             return verify.verify(signature);
         }
 
-        private void checkDigestLength(byte[] digest) {
-            if (digest.length != getDigestLength()) {
-                throw new IllegalArgumentException("Invalid digest length.");
-            }
+    }
+
+    private void checkDigestLength(byte[] digest) {
+        if (digest.length != getDigestLength()) {
```

Comment:
This was a false positive right ?

Reply:
Yes, however, we want to ensure our code is LTS compatible. So, if spotbugs check fails for JDK 11, then it's better to fix it for JDK8 too.

This is only to test that we get the same results in PR build as we get in post-merge CI builds.

Reply:
Looks like the PR build failed due to spotbugs. So, the fix is working!

Reply:
Yeah, I agree, this is a good idea to enable this in PR builds.
It would be good to catch false positives at this stage itself and either refactor to fix or suppress them, which ever is the best option.